cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/NamespaceHttpHandlerTest.java

```diff
@@ -277,6 +279,12 @@ public void testDeleteAll() throws Exception {
     Assert.assertTrue(streamAdmin.exists(myStream));
     assertResponseCode(200, deleteNamespace(OTHER_NAME));
     Assert.assertFalse(streamAdmin.exists(myStream));
+
+    assertResponseCode(200, createNamespace(OTHER_NAME));
```

Comment:
Ok, I probably missed line 258. Anyway, maybe we should add a comment on why we need to drop and recreate the namespace then.